src/Components/Home.js

```diff
@@ -2,21 +2,37 @@ import './Home.css';
 import { useState } from 'react';
 
 const Home = () => {
-    const [value, setValue] = useState('');
+    const initialValue = [];
+    const [value, setValue] = useState([]);
+    const [initial, setInitial] = useState(false);
     var val = '';
```

Comment:
why val is declared with 'var' ???
create a state variable for individual todo and use it as a value in in the input field. Read about Controlled Components